### DISCUSSION TITLE
using FilterProvider(@JsonFilter) with EMFJson

### DISCUSSION THREAD

**Discussion by pronithg** [eclipse-emfcloud/emfcloud#118] [EMFJson Jackson]
Hi,

How can I use ObjectMapper FilterProvider?

**Comment by martin-fleck-at** [comment]
Hi @pronithg, I do not think it is currently possible to use FilterProvider within EMFJson Jackson, at least not that I am aware of. I created a feature request to track this: https://github.com/eclipse-emfcloud/emfjson-jackson/issues/27. If anyone is willing to put in the work, we are happy to accept PRs for it.
